karton/dashboard/app.py

```diff
@@ -43,6 +39,21 @@ class KartonDashboard(KartonBase):
 
 karton = KartonDashboard()
 
+base_path = (
+    karton.config.get("dashboard", "base_path")
+    if karton.config.has_option("dashboard", "base_path")
+    else ""
+)
```

Comment:
Isnt this equivalent to just

```suggestion
base_path = karton.config.get("dashboard", "base_path") or ""
```
(note: i didn't test it in this case, but `None or "" == ""`)

Reply:
Actually it's even more simple
```suggestion
base_path = karton.config.get("dashboard", "base_path", fallback="")
```
See Config.get definition https://github.com/CERT-Polska/karton/blob/master/karton/core/config.py#L87